exercises/practice/phone-number/test/phone_number_test.cljs

```diff
@@ -27,7 +27,7 @@
   (is (= "987" (phone-number/area-code "(987) 654-3210"))))
 
 (deftest area-code-with-full-us-phone-number
-  (is (= "123" (phone-number/area-code "11234567890"))))
+  (is (= "223" (phone-number/area-code "12234567890"))))
 
 (deftest pretty-print
   (is (= "(123) 456-7890" (phone-number/pretty-print "1234567890"))))
```

Comment:
This one is incorrect as far as being invalid, so should not pretty print as it is.  (It should not get that far.)

Reply:
Yeah, there's something funky with the test definitions in general! I'll go through them and make it fully conform to the canonical-format! Might as well do it in one swoop while we're at it :)

Reply:
Thank you, I appreciate it!

Reply:
Just noticed the behavior for errors is actually wrong. It currently expects to return "0000000000" for invalid numbers, while the canonical file states that errors should be returned? Fixing that to expect the errors as well, I suppose? Btw, the same exercise in Clojure expects those zeroes instead of errors too iirc,.. 🤔

Reply:
What's our stand with errors? Should we test for only errors in general or test for error messages equivalent of the ones in canonical file?

Reply:
The track decides how to handle the error for the specific exercise (not sure what was decided here, but I would take the current test hints, as it was reviewed and brought in at some point) and I believe some tracks chose to raise an exception, while others chose to use all zeros.

Looking for ocnfirmation from the @exercism/clojurescript team.

Reply:
Alright great! I'll add the missing tests for now and keep the zeroes unless instructed otherwise! 👍

Reply:
BOOM! Previously missed tests added (and the pretty-print fixed you mentioned)!

Reply:
Thank you!  Now to wait for someone else to review.